src/lint/practice_exercises.nim

```diff
@@ -0,0 +1,64 @@
+import std/[json, os]
+import ".."/helpers
+import "."/validators
+
+proc isValidAuthorOrContributor(data: JsonNode, context: string, path: string): bool =
```

Comment:
This code is currently duplicated between the practice_exercises and concept_exercises modules. We could extract it to some shared module.